tests/test.py

```diff
@@ -10138,6 +10138,211 @@ class TestPostgresqlTypeAwareDecryptionWithDefaultsPsycopg3(Psycopg3ExecutorMixi
+
+    def testLowerConfigLowerQuotedQuery(self):
+        # should match, lowercase config identifier == lowercase SQL identifier
+        self.runTestCase("lowercase_table", True, "DATA", True, True)
+
+    def testLowerConfigUpperQuery(self):
+        # should match, lowercase config identifier == lowercase SQL identifier (converted)
+        self.runTestCase("LOWERCASE_TABLE", False, "DATA", True, True)
+
+    def testLowerConfigUpperQuotedQuery(self):
+        # should NOT match, lowercase config identifier != uppercase SQL identifier
+        self.runTestCase("LOWERCASE_TABLE", True, "DATA", True, False)
+
+    def testUpperConfigLowerQuery(self):
+        # should NOT match, uppercase config identifier != lowercase SQL identifier
+        self.runTestCase("uppercase_table", False, "data", False, False)
+
+    def testUpperConfigLowerQuotedQuery(self):
+        # should NOT match, uppercase config identifier != lowercase SQL identifier
+        self.runTestCase("uppercase_table", True, "data", False, False)
+
+    def testUpperConfigUpperQuery(self):
+        # should NOT match, uppercase config identifier != lowercase SQL identifier (converted)
+        self.runTestCase("UPPERCASE_TABLE", False, "data", False, False)
+
+    def testUpperConfigUpperQuotedQuery(self):
+        # should match, uppercase config identifier == uppercase SQL identifier
+        # column should match in all cases except quoted "DATA"
+        self.runTestCase("UPPERCASE_TABLE", True, "data", False, True)
+        self.runTestCase("UPPERCASE_TABLE", True, "data", True, True)
+        self.runTestCase("UPPERCASE_TABLE", True, "DATA", False, True)
+        self.runTestCase("UPPERCASE_TABLE", True, "DATA", True, False)
+
+
+class TestDifferentCaseTableIdentifiersMySQL(BaseTransparentEncryption):
+    ENCRYPTOR_CONFIG = get_encryptor_config('tests/encryptor_configs/mysql_identifiers.yaml')
+
+    def checkSkip(self):
+        if not TEST_WITH_TLS or not (TEST_MYSQL or TEST_MARIADB):
+            self.skipTest("this test is only for MySQL/MariaDB")
+
+    def setUp(self):
+        super().setUp()
+        self.engine1.execute(f"CREATE TABLE IF NOT EXISTS lowercase_table (id INT PRIMARY KEY AUTO_INCREMENT, data BLOB);")
+        self.engine1.execute(f"CREATE TABLE IF NOT EXISTS LOWERCASE_TABLE (id INT PRIMARY KEY AUTO_INCREMENT, data BLOB);")
+        self.engine1.execute(f"CREATE TABLE IF NOT EXISTS uppercase_table (id INT PRIMARY KEY AUTO_INCREMENT, data BLOB);")
+        self.engine1.execute(f"CREATE TABLE IF NOT EXISTS UPPERCASE_TABLE (id INT PRIMARY KEY AUTO_INCREMENT, data BLOB);")
+
+    def tearDown(self):
+        self.engine1.execute(f"DROP TABLE lowercase_table;")
+        self.engine1.execute(f"DROP TABLE LOWERCASE_TABLE;")
+        self.engine1.execute(f"DROP TABLE uppercase_table;")
+        self.engine1.execute(f"DROP TABLE UPPERCASE_TABLE;")
+        super().tearDown()
+
+    def runTestCase(self, table_name: str, quoted: bool, should_match: bool):
```

Comment:
can you add comment somewhere why we don't test here different cases for columns? will be much better to leave reference to doc with explanation that mysql doesn't have case-sensitivity for columns